Modules/Devices/src/Devices.Infrastructure.Database.Postgres/Migrations/20230703123830_PnsRegistration.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Devices.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class PnsRegistration : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "PnsRegistration",
+                columns: table => new
+                {
+                    Handle = table.Column<string>(type: "character varying(200)", maxLength: 200, nullable: false),
+                    IdentityAddress = table.Column<string>(type: "character(36)", unicode: false, fixedLength: true, maxLength: 36, nullable: true),
+                    DeviceId = table.Column<string>(type: "character(20)", unicode: false, fixedLength: true, maxLength: 20, nullable: true),
```

Comment:
These shouldn't be nullable.

In these cases you have to do it explicitly in the EntityTypeConfiguration, because the class is part of a project which doesn't have nullable reference types enabled.

Remember to not just update the migration files, but instead delete the migration and create a new one. Use the powershell scripts to delete migration.